src/amg/azext_amg/commands.py

```diff
@@ -32,6 +32,15 @@ def load_command_table(self, _):
         g.custom_command('query', 'query_data_source')
         g.custom_command('update', 'update_data_source')
 
+    with self.command_group('grafana notification-channel') as g:
+        g.custom_command('list', 'list_notification_channels')
+        g.custom_command('list-short', 'list_notification_channels_short')
```

Comment:
I suggest we merge this with `list` command and expose an argument of `--short`

Reply:
Valid suggestion. Applied in commit [REV1]